### PR TITLE
Adding contribution docs

### PR DESCRIPTION
**Pull request by DarrylBrooks97**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+labels: ["bug", "blog", "core"]
+assignees:
+  - gt-codes
+  - darrylbrooks97
+body:
+  - type: checkboxes
+    id: Code-of-conduct
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/)
```
Comment (gt-codes) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:13]: Can you create a page on our site and add the link here? Ex: `blacc.xyz/conduct`

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "💡 Feature request"
+labels: [💡 Feature Request]
+assignees:
+  - Garrett1Tolbert
```
Comment (gt-codes) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:6]:
```suggestion
  - gt-codes
```

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - DarrylBrooks97
+body:
+  - type: checkboxes
+    id: Code-of-conduct
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/)
```
Comment (gt-codes) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:13]: Same as above where this conduct needs to be on our site

**Review by gt-codes — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 1 above)
Comment (DarrylBrooks97) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: https://core-atbcwuf7k-blacc.vercel.app/conduct

**Pull request merged**